tests/test_auxiliary_functions.py

```diff
@@ -0,0 +1,20 @@
+"""
+Tests for polaraveraging.py
+"""
+import clmm
+import clmm.polaraveraging as pa
+from numpy import testing
+import numpy as np
+
+
+def test_theta_units_conversion():
+    # tests for invaid input
+    testing.assert_raises(ValueError, pa._theta_units_conversion, np.pi, 'crazy units')
+    testing.assert_raises(ValueError, pa._theta_units_conversion, np.pi, 'Mpc')
+    testing.assert_raises(ValueError, pa._theta_units_conversion, np.pi, 'Mpc', 0.5)
+
+    # tests for angular conversion
```

Comment:
We need to be sure to test every allowed conversion option. There were more than 4. For each, we should 
1. Test theta=0
2. Test theta=pi
3. Test theta=-pi
4. Test theta=48.48 (Just a random float)